docker-compose.yaml

```diff
@@ -0,0 +1,27 @@
+version: "3"
+services:
+  llama-cpp:
+    build:
+      context: .
```

Comment:
Podman build fails with
```
Traceback (most recent call last):
  File "/usr/bin/podman-compose", line 33, in <module>
    sys.exit(load_entry_point('podman-compose==1.1.0', 'console_scripts', 'podman-compose')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/podman_compose.py", line 3308, in main
    asyncio.run(async_main())
  File "/usr/lib64/python3.12/asyncio/runners.py", line 194, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/asyncio/base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/podman_compose.py", line 3304, in async_main
    await podman_compose.run()
  File "/usr/lib/python3.12/site-packages/podman_compose.py", line 1634, in run
    retcode = await cmd(self, args)
              ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/podman_compose.py", line 2314, in compose_up
    if await compose.commands["build"](compose, build_args) != 0:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/podman_compose.py", line 2271, in compose_build
    s = await t
        ^^^^^^^
  File "/usr/lib64/python3.12/asyncio/tasks.py", line 631, in _wait_for_one
    return f.result()  # May raise f.exception().
           ^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/podman_compose.py", line 2224, in build_one
    raise OSError("Dockerfile not found in " + ctx)
OSError: Dockerfile not found in /home/jpodivin/repos/logdetective
Exception ignored in: <function BaseSubprocessTransport.__del__ at 0x7f695ed8f880>
Traceback (most recent call last):
  File "/usr/lib64/python3.12/asyncio/base_subprocess.py", line 126, in __del__
  File "/usr/lib64/python3.12/asyncio/base_subprocess.py", line 104, in close
  File "/usr/lib64/python3.12/asyncio/unix_events.py", line 568, in close
  File "/usr/lib64/python3.12/asyncio/unix_events.py", line 592, in _close
  File "/usr/lib64/python3.12/asyncio/base_events.py", line 795, in call_soon
  File "/usr/lib64/python3.12/asyncio/base_events.py", line 541, in _check_closed
RuntimeError: Event loop is closed
Error: executing /usr/bin/podman-compose up: exit status 1
```

You should explicitly set the container file here [1]

[1]https://docs.docker.com/compose/compose-file/build/#dockerfile

Reply:
I need to try podman-compose